cmd/package_test.go

```diff
@@ -233,21 +227,13 @@ func (s *suite) TestRecursiveRunYamlsWithOwnRunYaml(c *C) {
 	// Prepare.
 	s.importFakeOSvBootstrapPkg(c)
 	s.importFakeDemoPkg(c)
-	packageYamlText := `
-name: package-name
-title: PackageTitle
-author: package-author
-require:
-  - fake.demo
-`
-	runYamlText := `
-runtime: native
-config_set:
-  ownBoot:
-    bootcmd: echo MyBoot
-`
-	ioutil.WriteFile(filepath.Join(s.packageDir, "meta", "package.yaml"), []byte(packageYamlText), 0700)
-	ioutil.WriteFile(filepath.Join(s.packageDir, "meta", "run.yaml"), []byte(runYamlText), 0700)
+	s.requireFakeDemoPkg(c)
+	s.setRunYaml(fixIndent(`
```

Comment:
Would you consider moving `fixIndent` to `setRunYaml`? Here, you would simply have `s.setRunYaml(...)` and the indents would be handled internally.